playground/infrastructure/ci_cd.py

```diff
@@ -16,37 +16,76 @@
 """
 Module implements CI/CD steps for Beam Playground examples
 """
-
+import argparse
```

Comment:
I don't see it in requirements, shouldn't this be added?

Reply:
This import from standard lib :)